test/client/app/components/higherOrder/hasConfig.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import hasConfig from '../../../../../assets/app/components/higherOrder/hasConfig';
+
+const mock = React.createClass({
```

Comment:
FWIW, since everything goes through the babel JSX pipeline (including tests), we should be able to write
`const mock = <div></div>;` instead of having to write the React class boilerplate.